examples/policies/bpf_program_attached.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: aquasecurity.github.io/v1beta1
+kind: TraceePolicy
```

Comment:
The apiVersion and kind will soon be changed (@josedonizetti ) so if this will be merged after, don't forget to update it as well